content/blog/posts/osl-staff-mentors-international-students-through-google-summer-of-code.rst

```diff
@@ -1,51 +1,90 @@
+
+
+  *“This allows system administrators to view the state and history of the
+  machines in their cluster in a very visual and intuitive way,” Lett says. “His
+  work also created an infrastructure on which further visualization and
+  analysis tools can be built. Writing flexible, extensible and pluggable code
+  not only makes our work easier in Ganeti Web Manager, but will hopefully
+  inform and improve his future work in any type of programming.”*
+
+.. image:: /theme/img/PiotrAtTheZoo.JPG
+  :scale: 100%
+  :alt: Piotr at the Zoo
+
+Working with developers more than 5,500 miles away from his hometown near
+Cracow, Poland, Banaszkiewicz says, was only difficult because of the nine-hour
+time difference that meant staff members didn’t arrive for work at the OSL
+office until it was evening in Poland. But by connecting with OSL staff over
+Internet Relay Chat and sharing ideas through Google Docs, he says was able to
+learn from his mentors at the OSL and improve his knowledge about virtualization
+and working with Python and Vagrant.
+
+  *“I can see the OSL as a very pro-student organization, which I was delighted
+  to work for,” he says. “There are some very passionate programmers, who
+  actually were like teachers for me during GSoC.”*
+
+Banaszkiewicz is unique among GSoC students the OSL has mentored in that he
+previously worked with the lab through Google Code-in, a program similar to GSoC
+but designed for high school students. He has been the only Google Code-in
+student to continue to be mentored by the OSL in GSoC. Performing tasks for the
+OSL as a high school student, Banaszkiewicz says, introduced him to the tools
+used in professional open source development.
+
+  *“I was a little younger and did not know many technologies used by the OSL
+  team,” he says. “Now these technologies are obligatory in any project, and I'm
+  very happy I could learn them so early.”*
+
+Now, Banaszkiewicz is studying at the AGH University of Science in Cracow. He
+plans to apply to work as a GSoC student with the OSL again this summer and
+continue to expand his knowledge of open source development. The OSL’s continued
+participation in GSoC, Lett says, not only offers students a unique opportunity,
+but also presents a host of benefits both for the lab and the open source
+community.
+
+  *“Mentoring young open source developers bolsters the community we rely on,
+  and helps make sure the new generation of open source developers comes in with
+  real skill and knowledge,” Lett says. “It also creates connections that bring
+  benefits back to us in the form of contributions to our projects,
+  collaborations with other projects and relationships with developers around
+  the world.”*
+
+Media contact: Kayla Harr, harrk@osuosl.org To learn more about the students who
+gain real-world experience through the Open Source Lab, visit our `staff`_ page.
+More information about Google Summer of Code and how to participate can be found
+`here`_.
```

Comment:
Let's not use generics like 'here' for named link references - I think "can be found at the `OSL GSOC page`_ or something like that.